src/ray/common/ray_config_def.h

```diff
@@ -328,6 +328,14 @@ RAY_CONFIG(bool, put_small_object_in_memory_store, false)
 /// pipelining task submission.
 RAY_CONFIG(uint32_t, max_tasks_in_flight_per_worker, 1)
 
+/// Whether work stealing among existing workers should be enabled.
+RAY_CONFIG(bool, work_stealing_enabled, false)
+
+/// Whether the eager worker requesting mode should be enabled, so that
+/// we can request new workers for the purpose of stealing work from overloaded
+/// existing workers.
+RAY_CONFIG(bool, work_stealing_and_eager_workers_requesting_enabled, false)
```

Comment:
After discussing this offline, we are keeping work stealing and eager worker requesting together, but we are adding only one boolean variable (instead of two) to the config files